torch/csrc/autograd/generated/variable_factories.h

```diff
@@ -388,17 +392,20 @@ inline at::Tensor bartlett_window(int64_t window_length, const at::TensorOptions
     node = tracer_state->graph->create(op_name, /*num_outputs=*/0);
     jit::tracer::recordSourceLocation(node);
     jit::tracer::addInputs(node, "window_length", window_length);
-    jit::tracer::addInputs(node, "options", options);
+    jit::tracer::addInputs(node, "dtype", dtype);
+    jit::tracer::addInputs(node, "layout", layout);
+    jit::tracer::addInputs(node, "device", device);
+    jit::tracer::addInputs(node, "pin_memory", pin_memory);
     tracer_state->graph->insertNode(node);
-  
+
     jit::tracer::setTracingState(nullptr);
   }
   at::Tensor tensor = ([&]() {
     at::AutoNonVariableTypeMode non_var_type_mode(true);
-    return at::bartlett_window(window_length, at::TensorOptions(options).is_variable(false));
+    return at::bartlett_window(window_length, dtype, layout, device, pin_memory);
   })();
   at::Tensor result =
-    autograd::make_variable(std::move(tensor), /*requires_grad=*/options.requires_grad());
+    autograd::make_variable(std::move(tensor), /*requires_grad=*/false);
```

Comment:
The JIT operates on variables. Because of that, there probably needs to be a `requires_grad` input to this function

Reply:
When i talked to @gchanan about this, the point was that `requires_grad` is not being set anywhere till this point and is generally a hack in some gen files. He told me to pass `false` in cases like this. Eventually (step 2-3 of the proposal) we will fix `requires_grad` and there wont be magical logic like this.

Reply:
Does this change break JIT tracer support for `barlett_window(..., requires_grad=True)`?